d2l-hm-constants-behavior.html

```diff
@@ -89,6 +89,12 @@
 				gateways: 'https://notifications.api.brightspace.com/rels/gateways',
 				gateway: 'https://notifications.api.brightspace.com/rels/gateway',
 				organizationNotifications: 'https://notifications.api.brightspace.com/rels/organization-notifications'
+			},
+			// Rubrics API sub-domain rels
+			Rubrics: {
+				criteriaGroups: 'https://rubrics.api.brightspace.com/rels/criteria-groups',
+				criteria: 'https://rubrics.api.brightspace.com/rels/criteria',
```

Comment:
The file isn't super consistent but it might be nice to swap `criteria` above `criteriaGroups` for alphabetical goodness.